src/Transition/InputRulesFiles/Rules8-7-0-to-8-8-0.md

```diff
@@ -0,0 +1,11 @@
+Input Changes
+=============
+
+This file documents the structural changes on the input of EnergyPlus that could affect interfaces, etc. 
+This was previously an Excel workbook that made for very difficult version control, especially during busy times around code freezes.
+
+# Object Change: `ZoneControl:ContaminantController`
+
+Insert new blank field 6 (A6) (for Maximum Carbon Dioxide Concentration Schedule Name). 
+Shift all later fields down by one.
+
```

Comment:
OK, I just added, and pushed, the transition rule for this.  I added it conditionally, so that the new field is only added **if** the object needs to have it added.  Min-fields is smaller, so if the object only has 4 fields, no need to add this one on the end.  I tested with objects of 4, and then 6 fields and the transition is working nicely.